src/main/kotlin/racingcar/model/Car.kt

```diff
@@ -1,6 +1,22 @@
 package racingcar.model
 
-class Car(var name: String, var position: String = "")
+const private val needCarLen="자동차 이름은 1자 이상 5자 이하이어야 합니다. 입력된 이름:"
```

Comment:
```suggestion
const private val needCarLen = "자동차 이름은 1자 이상 5자 이하이어야 합니다. 입력된 이름:"
```
커밋 작성전에 Android studio 에서 제공되는 reformat 기능을 통해 한번씩 포매팅 체크도 부탁드려요~!